Game/ConsoleApplication2/Constants.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <SFML\System\Vector2.hpp>
+
+
+class Constants
+{
+public:
+	static const sf::Vector2f UP;
+	static const sf::Vector2f LEFT;
+	static const sf::Vector2f DOWN;
```

Comment:
These should be static functions of either Vector2D, or they already exist in sf::Vector2f, not entirely sure, but this kind of constant should definitely be encapsulated by the vector class, not as a global constant